src/avfoundation.cs

```diff
@@ -14343,4 +14841,828 @@ public enum AVCaptureCenterStageControlMode : long {
+	[BaseType (typeof (AVDelegatingPlaybackCoordinatorPlaybackControlCommand))]
+	[DisableDefaultCtor]
+	interface AVDelegatingPlaybackCoordinatorBufferingCommand
+	{
+		[Export ("anticipatedPlaybackRate")]
+		float AnticipatedPlaybackRate { get; }
+
+		[NullAllowed, Export ("completionDueDate")]
+		NSDate CompletionDueDate { get; }
+	}
+
+	[TV (15,0), NoWatch, Mac (12,0), iOS (15,0), MacCatalyst (15,0)]
+	[BaseType (typeof (AVDelegatingPlaybackCoordinatorPlaybackControlCommand))]
+	[DisableDefaultCtor]
+	interface AVDelegatingPlaybackCoordinatorPauseCommand
+	{
+		[Export ("shouldBufferInAnticipationOfPlayback")]
+		bool ShouldBufferInAnticipationOfPlayback { get; }
+
+		[Export ("anticipatedPlaybackRate")]
+		float AnticipatedPlaybackRate { get; }
+	}
+
+	[TV (15,0), NoWatch, Mac (12,0), iOS (15,0), MacCatalyst (15,0)]
+	[BaseType (typeof (AVDelegatingPlaybackCoordinatorPlaybackControlCommand))]
+	[DisableDefaultCtor]
+	interface AVDelegatingPlaybackCoordinatorPlayCommand
+	{
+		[Export ("rate")]
+		float Rate { get; }
+
+		[Export ("itemTime")]
+		CMTime ItemTime { get; }
+
+		[Export ("hostClockTime")]
+		CMTime HostClockTime { get; }
+	}
+
+	[TV (15,0), NoWatch, Mac (12,0), iOS (15,0), MacCatalyst (15,0)]
+	[BaseType (typeof (NSObject))]
+	[DisableDefaultCtor]
+	interface AVDelegatingPlaybackCoordinatorPlaybackControlCommand
+	{
+		[NullAllowed, Export ("originator")]
+		AVCoordinatedPlaybackParticipant Originator { get; }
+
+		[Export ("expectedCurrentItemIdentifier")]
+		string ExpectedCurrentItemIdentifier { get; }
+	}
+
+	[TV (15,0), NoWatch, Mac (12,0), iOS (15,0), MacCatalyst (15,0)]
+	[BaseType (typeof (AVDelegatingPlaybackCoordinatorPlaybackControlCommand))]
+	[DisableDefaultCtor]
+	interface AVDelegatingPlaybackCoordinatorSeekCommand
+	{
+		[Export ("itemTime")]
+		CMTime ItemTime { get; }
+
+		[Export ("shouldBufferInAnticipationOfPlayback")]
+		bool ShouldBufferInAnticipationOfPlayback { get; }
+
+		[Export ("anticipatedPlaybackRate")]
+		float AnticipatedPlaybackRate { get; }
+
+		[NullAllowed, Export ("completionDueDate")]
+		NSDate CompletionDueDate { get; }
+	}
+
+	[TV (15,0), NoWatch, Mac (12,0), iOS (15,0), MacCatalyst (15,0)]
+	[BaseType (typeof (NSObject))]
+	[DisableDefaultCtor]
+	interface AVPlaybackCoordinator
+	{
+
+		[Notification]
+		[Field ("AVPlaybackCoordinatorOtherParticipantsDidChangeNotification")]
+		NSString OtherParticipantsDidChangeNotification { get; }
+
+		[Notification]
+		[Field ("AVPlaybackCoordinatorSuspensionReasonsDidChangeNotification")]
+		NSString SuspensionReasonsDidChangeNotification { get; }
+
+		[Export ("otherParticipants")]
+		AVCoordinatedPlaybackParticipant[] OtherParticipants { get; }
+
+		[Export ("suspensionReasons")]
+		string[] SuspensionReasons { get; }
+
+		[Export ("beginSuspensionForReason:")]
+		AVCoordinatedPlaybackSuspension BeginSuspension (string suspensionReason);
+
+		[Export ("expectedItemTimeAtHostTime:")]
+		CMTime ExpectedItemTimeAtHostTime (CMTime hostClockTime);
```

Comment:
```suggestion
		CMTime GetExpectedItemTime (CMTime hostClockTime);
```
Perhaps verb-ify and remove suffix